force-app/main/default/lwc/esa_LC_displayDECInformation/esa_LC_displayDECInformation.js

```diff
@@ -0,0 +1,24 @@
+import { LightningElement, api, wire } from 'lwc';
+import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+import ACCOUNT_NAME from '@salesforce/schema/Case.DEC__r.Name';
+import ACCOUNT_DEC from '@salesforce/schema/Case.DEC__r.DEC__c';
+//import labelName from '@salesforce/label/label-reference';
+
+const fields = [ACCOUNT_NAME,ACCOUNT_DEC];
+
+export default class Esa_LC_displayDECInformation extends LightningElement {
+    @api recordId;
+
+    @wire(getRecord, { recordId: '$recordId', fields })
+    case;
+
+    get accountName() {
+        return getFieldValue(this.case.data, ACCOUNT_NAME);
+    }
+
+    get accountDEC() {
+        return getFieldValue(this.case.data, ACCOUNT_DEC);
+    }
+        
```

Comment:
@deepakmulamalla: Please remove white-spaces in Apex and LWC when creating PRs. This ensures that your code is clean. Thanks.